src/normalize/core.ts

```diff
@@ -212,6 +211,13 @@ export class CoreNormalizer extends SpecMapper<NormalizerParams, FacetedUnitSpec
 
     return super.mapFacet(spec, params);
   }
+  //
+  private isEmpty(obj: any) {
```

Comment:
I think this should be a helper function outside of this class, no?

Reply:
Yes, move this outside the class